build/packages-template/bbb-graphql-server/after-install.sh

```diff
@@ -23,16 +23,18 @@ case "$1" in
 
   echo "Postgresql configured"
 
-  #Generate a random password to Hasura to improve security
-  HASURA_ADM_PASSWORD=$(grep '^HASURA_GRAPHQL_ADMIN_SECRET=' /etc/default/bbb-graphql-server | cut -d '=' -f 2)
-  if [ "$HASURA_ADM_PASSWORD" = "bigbluebutton" ]; then
-    echo "Set a random password to Hasura replacing the default 'bigbluebutton'"
-    HASURA_RANDOM_ADM_PASSWORD=$(openssl rand -base64 32 | sed 's/=//g' | sed 's/+//g' | sed 's/\///g')
-    sed -i "s/HASURA_GRAPHQL_ADMIN_SECRET=bigbluebutton/HASURA_GRAPHQL_ADMIN_SECRET=$HASURA_RANDOM_ADM_PASSWORD/g" /etc/default/bbb-graphql-server
-    HASURA_ADM_PASSWORD="$HASURA_RANDOM_ADM_PASSWORD"
-  fi
+#Generate a random password to Hasura to improve security
+if [ ! -f /usr/share/bbb-graphql-server/admin-secret.txt ]; then
+  mkdir -p /usr/share/bbb-graphql-server
+  HASURA_RANDOM_ADM_PASSWORD=$(openssl rand -base64 32 | sed 's/=//g' | sed 's/+//g' | sed 's/\///g')
+  echo "HASURA_GRAPHQL_ADMIN_SECRET=$HASURA_RANDOM_ADM_PASSWORD" > /usr/share/bbb-graphql-server/admin-secret.txt
+  chmod 644 /usr/share/bbb-graphql-server/admin-secret.txt
```

Comment:
Could we change this to 600? Only the "owner" to be able to read. The group, etc do not need access. Consider setting permissions for the directory too.

Something like:
`chown bigbluebutton:bigbluebutton /usr/share/bbb-graphql-server/admin-secret.txt`
`chmod 600 /usr/share/bbb-graphql-server/admin-secret.txt`
perhaps `chmod 700  /usr/share/bbb-graphql-server/`

Note that I have not tried this.